### ISSUE TITLE
[libtool] Remove --disable-static in build step

### ISSUE THREAD

**Issue by vincowl** [skycocker/chromebrew#3537]
Would it be possible to change 
```
system "./configure", "--disable-static", "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}"
``` 
to 
```
system "./configure", "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}"
```
in the 2.4.6-3 `libtool` package ? This breaks the new neovim 0.4.2 compilation. What would be the impact ?

Cf [https://github.com/neovim/neovim/issues/11198](https://github.com/neovim/neovim/issues/11198)

**Comment by uberhacker** [comment]
@vincowl: Feel free to submit a PR.  I think that would be a good thing to do and I don't anticipate any harmful side effects.  Just make sure the shared libraries are still generated after doing that.

**Comment by vincowl** [comment]
Fine ! How will we be sure to have no side effects since chromebrew don't seem to have something like a CI build that checks every package rebuild when one is modified ?

**Comment by uberhacker** [comment]
I'm fairly sure including extra files is fine.  The problems are introduced after removing file.  That is why I wanted you to check and make sure the shared libraries are not removed.

**Comment by vincowl** [comment]
Normally shared libraries will not be removed since `build_libtool_libs` is still set to `yes` once built. Is that enough or should I check something else ? Building neovim 0.4.2 from sources succeeds and it uses static and shared libs

**Comment by cstrouse** [comment]
@uberhacker @vincowl Is there a PR forthcoming for adding additional libraries for `libtool`? If not, I'll put one together and do some testing to make sure it doesn't cause any problems.

**Comment by uberhacker** [comment]
Not that I'm aware.  Feel free to go for it.

**Comment by vincowl** [comment]
No problem here too. We are facing a problem with neovim's compilation that may be related to libtool and `fstack-protector` compilation option. If you have any idea since you seem to have good knowledge in `libtool` and so on...  Feel free to have a look [here](https://github.com/neovim/neovim/issues/11228)

**Comment by cstrouse** [comment]
Closed by #3566.

**Comment by vincowl** [comment]
I have just tried a fresh install. I am facing the followinf problem when install script installs `libtool`. It cannot find `libfl.so`

```
make[1]: Entering directory '/usr/local/tmp/crew/libtool-2.4.6.tar.gz.dir/libtool-2.4.6'
Making all in .
make[2]: Entering directory '/usr/local/tmp/crew/libtool-2.4.6.tar.gz.dir/libtool-2.4.6'
  CC       libltdl/loaders/dlopen.lo
  CC       libltdl/lt__strl.lo
  CC       libltdl/loaders/libltdl_libltdl_la-preopen.lo
  CC       libltdl/libltdl_libltdl_la-lt__alloc.lo
  CC       libltdl/libltdl_libltdl_la-lt_dlloader.lo
  CC       libltdl/libltdl_libltdl_la-lt_error.lo
  CC       libltdl/libltdl_libltdl_la-ltdl.lo
  CC       libltdl/libltdl_libltdl_la-slist.lo
  CCLD     libltdl/dlopen.la
ar: error while loading shared libraries: libfl.so.2: cannot open shared object file: No such file or directory
make[2]: *** [Makefile:1112: libltdl/dlopen.la] Error 127
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/usr/local/tmp/crew/libtool-2.4.6.tar.gz.dir/libtool-2.4.6'
make[1]: *** [Makefile:1606: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/libtool-2.4.6.tar.gz.dir/libtool-2.4.6'
make: *** [Makefile:995: all] Error 2
libtool failed to install: `make V=0 -j2` exited with 2
```
